spec/datadog/appsec/contrib/active_record/patcher_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require 'datadog/appsec/spec_helper'
+require 'datadog/appsec/contrib/active_record/patcher'
+
+RSpec.describe Datadog::AppSec::Contrib::ActiveRecord::Patcher do
+  describe '#prepended_class_name' do
```

Comment:
Technically this is an internal method (which I hope can just go) and should not be tested directly because it repeats the next section of tests